source/get-started/bare-metal-install-server.rst

```diff
@@ -318,8 +318,10 @@ boot partition
 
 Now follow the same process to configure the remaining partitions.
 
-swap partition
---------------
+swap partition (optional)
+-------------------------
+
+A swapfile is generated by default during installation. However, if you wish to manually create a swap partition, follow the steps below. 
```

Comment:
```suggestion
A swapfile is generated by default during installation. However, if you instead wish to create a swap partition, follow the steps below. 
```

Reply:
Same comment as above for `-desktop` version.